src/login.tsx

```diff
@@ -19,6 +21,14 @@ const Login = () => {
   const [password, setPassword] = useState<string>('');
   const [emailError, setEmailError] = useState<FieldErrors>();
   const [passwordError, setPasswordError] = useState<FieldErrors>();
+  const [loginError, setLoginError] = useState<any>('');
```

Comment:
```suggestion
  const [loginError, setLoginError] = useState('');
```
quando você seta um valor inicial pro useState, o ts já infere o tipo

Reply:
Ah, o any é resquício de quando eu tentei usar essa variável pra guardar o objeto completo do usuário retornado pelo servidor, mudei o propósito da variável mas esqueci de mudar isso. Sei que ele é capaz de inferir o tipo, mas é uma boa prática manter ele sempre tipado né?

Reply:
aqui na taq não temos um padrão em relação a isso. e apesar de ter te falado da inferência do ts, eu prefiro tipar também kkk
mas normalmente vai de projeto para projeto 👍